FeBuddyLibrary/Helpers/DownloadHelpers.cs

```diff
@@ -0,0 +1,107 @@
+﻿using FeBuddyLibrary.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FeBuddyLibrary.Helpers
+{
+    public class DownloadHelpers
+    {
+        private static string TempPath { get; } = GlobalConfig.tempPath;
```

Comment:
Can't use properties or variables here because the class is technically not instantiated. Go back to using GlobalConfig.tempPath...